stable/datadog/templates/daemonset.yaml

```diff
@@ -12,6 +12,9 @@ spec:
         app: {{ template "fullname" . }}
       name: {{ template "fullname" . }}
     spec:
+      hostNetwork: true
```

Comment:
this is not needed:
https://github.com/kubernetes/heapster/issues/1586
I'll remove it.